Sources/LiveViewNative/Coordinators/LiveSessionCoordinator.swift

```diff
@@ -585,22 +584,50 @@ extension LiveSessionCoordinator {
 
     private static func getTarget() -> String {
```

Comment:
I updated this to send OS-style names. The only difference between the `target` and `os` arguments is now iPads will return `iPadOS` for the `target`, but `iOS` for the `os`.

Reply:
Can you ensure the target names are downcased? I'm up for debating this as we've preserved the upstream casing of the client's UI framework, but in this case within Phoenix the template names and filename segments have always been downcased.

Reply:
Sure, but I think the server may want to check for any casing, even something weird like `app.swiftui+IpAdOs.neex`. Not all filesystems are case sensitive like APFS though.

Reply:
The issue isn't the filesystem, the issue is Elixir's own function file finding files by glob: https://hexdocs.pm/elixir/Path.html#wildcard/2 is case-sensitive. It's used in `Phoenix.Template` https://github.com/phoenixframework/phoenix_template/blob/main/lib/phoenix/template.ex#L352 for finding the relevant template and I'm leaning into that implementation